python/astro_metadata_translator/cli/astrometadata.py

```diff
@@ -26,10 +26,11 @@
 PACKAGES_VAR = "METADATA_TRANSLATORS"
 
 hdrnum_option = click.option("-n", "--hdrnum",
-                             default=1,
+                             default=-1,
                              help="HDU number to read. If the HDU can not be found, a warning is issued but"
                              " reading is attempted using the primary header. The primary header is"
-                             " always read and merged with this header.")
+                             " always read and merged with this header. Negative number indicates that"
+                             " the second header will be merged if the FITS file supports extended FITS.")
```

Comment:
Perhaps mention that this is the default? I don't think Click output includes the default argument...